types/options.d.ts

```diff
@@ -43,7 +43,7 @@ interface AnnotationCoordinates {
   yMin?: Scriptable<ScaleValue, PartialEventContext>,
 }
 
-interface AnnotationPointCoordinates {
+interface AnnotationPointCoordinates extends AnnotationCoordinates{
```

Comment:
Missing space between `AnnotationCoordinates` and `{`
Should take another look at the linter, it should catch this kind of stuff.

Reply:
added blank before {